react/features/base/tracks/middleware.js

```diff
@@ -143,6 +143,10 @@ MiddlewareRegistry.register(store => next => action => {
         if (typeof APP !== 'undefined') {
             const result = next(action);
 
+            if (isPrejoinPageVisible(store.getState())) {
```

Comment:
why is this change necessary?

Reply:
not applicable anymore after removing the call to `getAvailableDevices`